proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,393 @@
+syntax = "proto3";
+
+package redpanda.api.dataplane.v1alpha2;
+
+import "buf/validate/validate.proto";
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message MountTopicsRequest {
```

Comment:
not needed strictly for this PR, might be good to get the API docs reviewed by docs team and improve or complete out the API documentation as needed.